workspaces/jenkins/plugins/jenkins-backend/src/service/jenkinsApi.ts

```diff
@@ -344,15 +344,8 @@ export class JenkinsApiImpl {
 
   async getJobBuilds(jenkinsInfo: JenkinsInfo, jobFullName: string) {
     let jobName = jobFullName;
```

Comment:
[Memory Allocation in Javacript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Memory_management) backs up my comment.

Reply:
Marking it as done! I did not catch that you are using the variable below if unchanged

Reply:
It will not as the url  is build using the jobName 
`${
        jenkinsInfo.baseUrl
      }/job/${jobName}/api/json?tree=${JenkinsApiImpl.jobBuildsTreeSpec.replace(
        /\s/g,
        '',
      )}`
      
      and jobName value changes only if jobFullName contains a / (job depth)
      
      even if we initialise it without value , we need to add else part stating jobname = jobFullName